testhelpers/docker_registry.go

```diff
@@ -119,6 +126,11 @@ func Custom(handler http.Handler, permissions map[string]ImagePrivileges) http.H
 	})
 }
 
+// Start creates a docker registry following these rules:
+// - Shared handler will be use, otherwise a new one will be created
+// - by default the shared handler will be wrapped using a read only handler
```

Comment:
```suggestion
// - By default the shared handler will be wrapped using a read only handler
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1a4373ac85d225b6bd3771c3ccadae2a239cf6db)